### ISSUE TITLE
SelectedObject's properties arent visible at all

### ISSUE THREAD

**Issue by artizzq** [bodong1987/Avalonia.PropertyGrid#16]
I have latest Avalonia 11.0.6 and your latest package installed.
My Selected object has bunch of properties
![image](https://github.com/bodong1987/Avalonia.PropertyGrid/assets/56273839/ba0f2048-f81f-4b00-b443-724135135151)
I set my object in XAML 
![image](https://github.com/bodong1987/Avalonia.PropertyGrid/assets/56273839/2eb9adee-63cd-4c98-972e-9cf71b0c3ba3)
And I see that in code my object is OK
![image](https://github.com/bodong1987/Avalonia.PropertyGrid/assets/56273839/5fd18252-b8ad-4a7a-a0a7-92c5d4efa720)
But when I see my GUI I dont see my properties.
![image](https://github.com/bodong1987/Avalonia.PropertyGrid/assets/56273839/2232c424-b29f-4cca-a7bb-dac72086f8b6)
Please clarify what details I need provide also to solve the problem...

**Comment by bodong1987** [comment]
> I have latest Avalonia 11.0.6 and your latest package installed. My Selected object has bunch of properties ![image](https://private-user-images.githubusercontent.com/56273839/289504812-ba0f2048-f81f-4b00-b443-724135135151.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNDgzNDYsIm5iZiI6MTcwMjM0ODA0NiwicGF0aCI6Ii81NjI3MzgzOS8yODk1MDQ4MTItYmEwZjIwNDgtZjgxZi00YjAwLWI0NDMtNzI0MTM1MTM1MTUxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEyVDAyMjcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ1Y2Y3YWNiYjhjZTMxNjdmNjdiOGY4ZjMyMDZkZmYyMGM0YmQ3ZmZmYmFhYWQzMzgyMjUyYjE4YTY3NzlhMmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kTTKcWpDOFK6CKggVwyZotBZTn3HItMO_rtXUnITRgs) I set my object in XAML ![image](https://private-user-images.githubusercontent.com/56273839/289504996-2eb9adee-63cd-4c98-972e-9cf71b0c3ba3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNDgzNDYsIm5iZiI6MTcwMjM0ODA0NiwicGF0aCI6Ii81NjI3MzgzOS8yODk1MDQ5OTYtMmViOWFkZWUtNjNjZC00Yzk4LTk3MmUtOWNmNzFiMGMzYmEzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEyVDAyMjcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJhYTIxMWZmMzhjM2E5MmFiMjA0OTdkMmE0ZTc0NTI4MzJkODI4NGE4Njg5Y2M3NzZmZDg5NTc1NDczMmViOTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.k5a6xMuGIpmzO4B4cerNztzJPI74qdURjkZeWuggS4c) And I see that in code my object is OK ![image](https://private-user-images.githubusercontent.com/56273839/289505367-5fd18252-b8ad-4a7a-a0a7-92c5d4efa720.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNDgzNDYsIm5iZiI6MTcwMjM0ODA0NiwicGF0aCI6Ii81NjI3MzgzOS8yODk1MDUzNjctNWZkMTgyNTItYjhhZC00YTdhLWEwYTctOTJjNWQ0ZWZhNzIwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEyVDAyMjcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBlY2RlNTQ2ZjU1NDE2ODhjOGRlOTQyZWJlMmFhM2I5YmUxMDkxZjc2ZGVkYjk2NjY2MzgzYTdjNGM1NGQxOTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-6utj-cp_aENqY2ri4pEzC_bbwGDJMjFoguIwBPLV-E) But when I see my GUI I dont see my properties. ![image](https://private-user-images.githubusercontent.com/56273839/289505827-2232c424-b29f-4cca-a7bb-dac72086f8b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNDgzNDYsIm5iZiI6MTcwMjM0ODA0NiwicGF0aCI6Ii81NjI3MzgzOS8yODk1MDU4MjctMjIzMmM0MjQtYjI5Zi00Y2NhLWE3YmItZGFjNzIwODZmOGI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEyVDAyMjcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwYmUwNDMwYTU2MDkyZDZiMDdiMzM3N2I1OTQ3YjI0OGMxZjMzMDliODg1M2ZiYmU0MjlmNTFmNDYxMGMwNzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rQR45kd7t0ZJ3eqiiFFoKcufhWGUuLNAc6m4DEybBks) Please clarify what details I need provide also to solve the problem...

If you want to display the properties of a custom object, you should choose to use ICustomTypeDescriptor and override its interface GetProperties, returning your PropertyDescriptor array in this interface. You can find corresponding examples in the sample ScriptableOptions.cs.

If my understanding is wrong, can you provide a streamlined demo project? I can debug directly.

**Comment by artizzq** [comment]
Okay, the problem lied inside one of the properties that was returning bad value. Exactly the one with DateTime. It's Kind wasnt specified (Needed to be UTC). And this caused to broke the entire PropertyGrid.

When I stored PropertyGrid inside the UserControl I havent had any exception (probably they are supressed somehow by Avalonia) and when I tried PropertyGrid inside the window I got an exception that was causing the problem, resulting no properties visible.

I guess it's solved, thanks for help!
